src/parse.rs

```diff
@@ -0,0 +1,159 @@
+use nom::character::complete::{one_of, char, multispace1, u8, u64};
+use nom::bytes::complete::tag;
+use nom::branch::alt;
+use nom::combinator::{recognize, opt};
+use nom::sequence::{tuple, terminated, preceded, pair};
+use nom::IResult;
+use nom::multi::{count, many1};
+
+#[derive(PartialEq, Debug)]
+#[allow(dead_code)]
```

Comment:
This is okay for now, but good to note that we shouldn't be allowing things like this in the future.